### PR TITLE
syscall/libc: add more syscall/libc symbols into csv file

### PR DESCRIPTION
**Pull request by anchao**
## Summary

syscall/libc: add more syscall/libc symbols into csv file

## Impact

N/A

## Testing

ci-check

### CODE REVIEW

**Review by acassis — APPROVED** [review]

**Comment by acassis** [comment]
What is this error?
```
wraps/WRAP_epoll_create1.c: In function '__wrap_epoll_create1':
Error: wraps/WRAP_epoll_create1.c:20:28: error: 'SYS_epoll_create1' undeclared (first use in this function); did you mean 'epoll_create1'?
   20 |   sched_note_syscall_enter(SYS_epoll_create1, 1, (uintptr_t)parm1);
      |                            ^~~~~~~~~~~~~~~~~
      |                            epoll_create1
wraps/WRAP_epoll_create1.c:20:28: note: each undeclared identifier is reported only once for each function it appears in
make[1]: *** [Makefile:70: WRAP_epoll_create1.o] Error 1
wraps/WRAP_epoll_ctl.c: In function '__wrap_epoll_ctl':
Error: wraps/WRAP_epoll_ctl.c:20:28: error: 'SYS_epoll_ctl' undeclared (first use in this function); did you mean 'epoll_ctl'?
   20 |   sched_note_syscall_enter(SYS_epoll_ctl, 4, (uintptr_t)parm1, (uintptr_t)parm2, (uintptr_t)parm3, (uintptr_t)parm4);
      |                            ^~~~~~~~~~~~~
      |                            epoll_ctl
wraps/WRAP_epoll_ctl.c:20:28: note: each undeclared identifier is reported only once for each function it appears in
make[1]: *** [Makefile:70: WRAP_epoll_ctl.o] Error 1
wraps/WRAP_epoll_wait.c: In function '__wrap_epoll_wait':
Error: wraps/WRAP_epoll_wait.c:20:28: error: 'SYS_epoll_wait' undeclared (first use in this function); did you mean 'epoll_wait'?
   20 |   sched_note_syscall_enter(SYS_epoll_wait, 4, (uintptr_t)parm1, (uintptr_t)parm2, (uintptr_t)parm3, (uintptr_t)parm4);
   ```

**Comment by anchao** [comment]
> What is this error?

@acassis 
I forgot to update `syscall_lookup.h`, now fixed

File: syscall/syscall.csv [hunk 1]
```diff
@@ -155,6 +166,8 @@
 "shmget","sys/shm.h","defined(CONFIG_MM_SHM)","int","key_t","size_t","int"
 "shutdown","sys/socket.h","defined(CONFIG_NET)","int","int","int"
 "sigaction","signal.h","","int","int","FAR const struct sigaction *","FAR struct sigaction *"
+"signal","signal.h","","_sa_handler_t","int","_sa_handler_t"
+"signalfd","sys/signalfd.h","","int","int","FAR const sigset_t *","int"
```
[the remark this answers is not in the document]
Comment (anchao) [reply syscall/syscall.csv]: Done! Thanks!

File: include/sys/syscall_lookup.h [hunk 2]
```diff
@@ -380,3 +380,34 @@ SYSCALL_LOOKUP(munmap,                     2)
...
+SYSCALL_LOOKUP(time,                       1)
+SYSCALL_LOOKUP(gettimeofday,               2)
+SYSCALL_LOOKUP(settimeofday,               2)
+
+/* ANSI C signal handling */
+
+SYSCALL_LOOKUP(signal,                     2)
+
+#ifdef CONFIG_SIGNAL_FD
+  SYSCALL_LOOKUP(signalfd,                 3)
```
Comment (pkarashchenko) [inline include/sys/syscall_lookup.h:412]: Let's group this with timerfd and eventfd that are grouped around the poll

Comment (anchao) [reply include/sys/syscall_lookup.h:412]: Done!

File: include/stdlib.h [hunk 3]
```diff
@@ -286,6 +286,11 @@ FAR void  *bsearch(FAR const void *key, FAR const void *base, size_t nel,
 
 FAR const char *getprogname(void);
 
+/* Registers a destructor function to be called by exit() */
+
+int __cxa_atexit(CODE void (*func)(FAR void *), FAR void *arg,
```
[the remark this answers is not in the document]
Comment (anchao) [reply include/stdlib.h:291]: But currently __cxa_atexit is implemented in libs/libc/stdlib/lib_atexit.c, __cxa_atexit is related to the NuttX system, In terms of the current directory structure stdlib.h is a more suitable location.

[the remark this answers is not in the document]
Comment (pkarashchenko) [reply include/stdlib.h:291]: Ok. Let's keep it for now in stdlib.h and refine in the future

**Comment by pkarashchenko** [comment]
@acassis please review and merge if no more comments

**Comment by anchao** [comment]
@acassis could you please review further?

**Comment by anchao** [comment]
@acassis ?

**Comment by pkarashchenko** [comment]
@masayuki2009 could you please take a look?

**Pull request merged**